### PR TITLE
Add `urdf_parser_py` fallback import

### PR DESCRIPTION
**Pull request by luisenp**

### CODE REVIEW

File: torchkin/torchkin/third_party/urdf_parser_py/sdf.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import torchkin.third_party.urdf_parser_py.xml_reflection as xmlr
+from torchkin.third_party.urdf_parser_py.xml_reflection.basics import *
+
+# What is the scope of plugins? Model, World, Sensor?
```
Comment (fantaosha) [inline torchkin/torchkin/third_party/urdf_parser_py/sdf.py:4]: Do need to add a license?

**Review by fantaosha — APPROVED** [review]
LGTM. Please confirm if we need to add a license for 3rd-parties?

File: torchkin/torchkin/third_party/urdf_parser_py/display_urdf.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+                        default=None, help='Dump file to XML')
+    args = parser.parse_args()
+
+    robot = URDF.from_xml_string(args.file.read())
+
+    print(robot)
+
+    if args.output is not None:
+        args.output.write(robot.to_xml_string())
```
Comment (mhmukadam) [inline torchkin/torchkin/third_party/urdf_parser_py/display_urdf.py:20]: Missing new line

**Review by mhmukadam — APPROVED** [review]

File: torchkin/torchkin/third_party/urdf_parser_py/display_urdf.py (hunk 2 above)
Comment (luisenp) [reply torchkin/torchkin/third_party/urdf_parser_py/display_urdf.py:20]: I prefer leaving this code as is.

**Review by fantaosha — APPROVED** [review]
LGTM

**Pull request merged**